Dockerfile

```diff
@@ -1 +1,34 @@
+FROM python:3.8.1
 
+ARG USE_REDIS
+ENV USE_REDIS=$USE_REDIS
+
+RUN apt-get -y update && apt-get -y upgrade && \
+    apt-get install --no-install-recommends -y wait-for-it postgresql-client ${USE_REDIS:+redis-tools}
+
+WORKDIR /tmp
+ARG REPOSITORY_URL=https://github.com/OpenSlides/openslides-datastore-service.git
+ARG GIT_CHECKOUT=master
+
+ARG MODULE
+RUN test -n "$MODULE" || (echo "MODULE not set" && false)
+ENV MODULE=$MODULE
+ARG PORT
```

Comment:
Please move this right before the EXPOSE statement. You do not have to clone, ... the repo, if you just change the port.